code/src/containers/forms/product-form/hooks/use-update-product/useUpdateProduct.test.ts

```diff
@@ -0,0 +1,86 @@
+import { renderHook } from "@testing-library/react";
+
+import useUpdateProduct from "@/containers/forms/product-form/hooks/use-update-product/useUpdateProduct";
+
+import routes from "@/constants/routes";
+
+const mockUnwrap = jest.fn();
+const mockNavigate = jest.fn();
+const mockOpenSnackbarWithTimeout = jest.fn();
+const mockUpdateProduct = jest.fn(() => ({ unwrap: mockUnwrap }));
+
+jest.mock("react-router-dom", () => ({
+  useNavigate: () => mockNavigate
+}));
+
+jest.mock("@/hooks/use-snackbar/useSnackbar", () => ({
+  __esModule: true,
+  default: () => ({
+    openSnackbarWithTimeout: mockOpenSnackbarWithTimeout
+  })
+}));
+
+jest.mock("@/store/api/productsApi", () => ({
+  useUpdateProductMutation: () => [mockUpdateProduct, {}]
+}));
+
+const successSnackbarConfig = {
+  variant: "success",
+  messageTranslationKey: "productForm.updation.success"
+};
+
+const errorSnackbarConfig = {
+  variant: "error",
+  messageTranslationKey: "productForm.updation.fail"
+};
+
+describe("Test useUpdateProduct hook", () => {
+  test("Should return correct values", () => {
+    const { result } = renderHook(() => useUpdateProduct());
```

Comment:
Use `beforeEach`

Reply:
I guess it is not the case for beforeEach because I need to use mock in one test, and that mock must not be used in other tests. I could create some render function, but I think there is no much sense in it when we have only one simple line